spark/src/test/scala/geotrellis/spark/io/index/zcurve/ZSpaceTimeKeyIndexSpec.scala

```diff
@@ -8,6 +8,7 @@ import scala.collection.immutable.TreeSet
 class ZSpaceTimeKeySpec extends FunSpec with Matchers{
   val y2k = new DateTime(2000, 1, 1, 0, 0)
   val upperBound = 8
+  val ymPattern = "YM"
```

Comment:
Usually we have test-specific vals inside the test itself, can you move this into the the `it` block?